internal/service/s3control/multi_region_access_point_data_source.go

```diff
@@ -0,0 +1,92 @@
+package s3control
+
+import (
+	"context"
+	"log"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/s3control"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-provider-aws/internal/conns"
+	"github.com/hashicorp/terraform-provider-aws/internal/verify"
+)
+
+func DataSourceMultiRegionAccessPoint() *schema.Resource {
+	return &schema.Resource{
+		ReadWithoutTimeout: dataSourceMultiRegionAccessPointBlockRead,
+
+		Schema: map[string]*schema.Schema{
+			"account_id": {
+				Type:         schema.TypeString,
+				Optional:     true,
+				ValidateFunc: verify.ValidAccountID,
+			},
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"alias": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"created_at": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"public_access_block": {
+				Type:     schema.TypeMap,
```

Comment:
I believe `public_access_block` contains a few more attributes, such as: `block_public_acls`, `block_public_policy`, etc.. 

You can change the type to a `schema.TypeList` too.

Reply:
Good point! 
Implemented it